### PR TITLE
Create history.md

### LINKED ISSUE #29: history
I'll be doing this command

### PR DESCRIPTION
**Pull request by skotov**
# Advanced HW 9 submission

Closes #29 

Name: Sophia Kotov
Uniqname: kotova

### CODE REVIEW

**Comment by mterwill** [comment]
You'll need to add a `<!--more-->` tag. See other merged files for an example.

File: _commands/basics/history.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+#### `history | grep <search_string>`
+This option allows you to search for most recent commands containing the search string
+~~~ bash
+$ history | grep cd
+    9  cd /home/name/secrets
+   11  cd topSecrets
+   14  history | grep cd
+~~~
+#### `!<string>`
```
Comment (mterwill) [inline _commands/basics/history.md]: Add an example above this showing `!11`

File: _commands/basics/history.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#### `history | grep <search_string>`
+This option allows you to search for most recent commands containing the search string
+~~~ bash
+$ history | grep cd
+    9  cd /home/name/secrets
+   11  cd topSecrets
+   14  history | grep cd
+~~~
+#### `!11`
```
Comment (mterwill) [inline _commands/basics/history.md]: Sorry if this was ambiguous – `!<#>` will re-run that command from your history. So in your case, `!11` would `cd topSecrets`.

**Comment by themoh** [comment]
We'll be closing all pull requests at midnight tonight, so please have this done by then.

**Comment by themoh** [comment]
You're having some formatting issues:

screen shot 2016-03-27 at 10 28 51 pm


Look at what the spacing around the lines with tilde's looks like in other files that've been merged in.

**Comment by mterwill** [comment]
Great!

**Pull request merged**